src/components/features/most-popular-section/tabs.tsx

```
@@ -6,65 +6,67 @@ import ProductItem from "../application/products/product-item";
+                {topFour.map((occasion) => (
+                    <TabsTrigger
+                        key={occasion._id}
+                        value={occasion._id}
+                        className="p-1.5 text-base font-medium text-zinc-700 data-[state=active]:text-maroon-600 data-[state=active]:shadow-none dark:text-zinc-400 dark:data-[state=active]:text-softpink-200"
+                    >
+                        {occasion.name}
+                    </TabsTrigger>
+                ))}
+            </TabsList>
+            {/* loop through the top occasions and render a tab for each one with its products */}
+            {productsByOccasion.map(({ _id, name, products }) => (
```

ESLint

[error] 'name' is defined but never used.

(@typescript-eslint/no-unused-vars)